### PR TITLE
droidmedia: Add function to query if codec is supported.

### PR DESCRIPTION
**Pull request by mlehtima**
[droidmedia] Add function to query is codec is supported. JB#47657

### CODE REVIEW

File: droidmediacodec.cpp [hunk 1]
```diff
@@ -508,6 +508,27 @@ class DroidMediaCodecBuilder {
     }
   }
 
+  static uint32_t flags(DroidMediaCodecMetaData *meta, uint32_t currentFlags) {
```
Comment (abranson) [inline droidmediacodec.cpp]: Why is this moved?

Comment (mlehtima) [reply droidmediacodec.cpp]: The function was private and I wanted to reuse it so I moved it and made it public and static.
